runtime/verbose/module.xml

```diff
@@ -102,22 +102,22 @@
 			<include path="$(OMR_DIR)/compiler" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/base" type="relativepath">
+			<include path="../compiler/base" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/ibm/base" type="relativepath">
+			<include path="../compiler/ibm/base" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source" type="relativepath">
+			<include path="../compiler" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/ibm" type="relativepath">
+			<include path="../compiler/ibm" type="relativepath">
```

Comment:
This isn't a path that exists even in the current source.  I think it can be deleted.

Reply:
Fixed in 626683a